docs/how-to/h-async-replication.md

```diff
@@ -0,0 +1,321 @@
+# Configuring asynchronous replication: MySQL InnoDB ClusterSet
+
+> **WARNING**: it is an internal article. Do NOT use it in production! Contact [Canonical Data Platform team](https://chat.charmhub.io/charmhub/channels/data-platform) if you are interested in the topic.
+
+## Intro
+Read more about [MySQL InnoDB ClusterSet](https://dev.mysql.com/doc/mysql-shell/8.0/en/innodb-clusterset.html) and watch the [presentation](https://videohub.oracle.com/media/Disaster+Recovery+with+MySQL+InnoDB+ClusterSet+-+What+is+it+and+how+do+I+use+itF/1_w0naxll1/170883371).
+
+For simplicity, we deploy both MySQL InnoDB Clusters into the same model, but the proper design is to install them in different Regions/Clouds:
+
+![MySQL_async_ClusterSet|690x586](upload://t8RCVAmkq1Hp5BwJM85ZJwxkREP.png) 
+
+## Bootstrap
+Bootstrap two MySQL InnoDB Clusters (Rome and Lisbon) and test application to generate SQL traffic:
+```shell
+juju deploy --constraints mem=2G mysql mysql-1st --channel 8.0/edge -n 3 --config cluster-name=ROME
+juju deploy --constraints mem=2G mysql mysql-2nd --channel 8.0/edge -n 3 --config cluster-name=LISBON
+juju deploy mysql-test-app --channel edge
+juju relate mysql-1st mysql-test-app:database
+juju wait-for application mysql-1st --query='status=="active"' && juju wait-for application mysql-2nd --query='status=="active"'
+```
+## Collect clusters details
+Collect both clusters credentials to reuse later:
+```shell
+cluster_name_1st=$(juju show-unit mysql-1st/0 | yq '.. | select(. | has("cluster-name")).cluster-name')
+cluster_name_2nd=$(juju show-unit mysql-2nd/0 | yq '.. | select(. | has("cluster-name")).cluster-name')
+root_password_1st=$(juju run-action mysql-1st/0 get-password username=root --wait | yq '.. | select(. | has("password")).password')
+root_password_2nd=$(juju run-action mysql-2nd/0 get-password username=root --wait | yq '.. | select(. | has("password")).password')
+serverconfig_password_1st=$(juju run-action mysql-1st/0 get-password username=serverconfig --wait | yq '.. | select(. | has("password")).password')
+serverconfig_password_2nd=$(juju run-action mysql-2nd/0 get-password username=serverconfig --wait | yq '.. | select(. | has("password")).password')
+clusteradmin_password_1st=$(juju run-action mysql-1st/0 get-password username=clusteradmin --wait | yq '.. | select(. | has("password")).password')
+clusteradmin_password_2nd=$(juju run-action mysql-2nd/0 get-password username=clusteradmin --wait | yq '.. | select(. | has("password")).password')
+
```

Comment:
Are monitoring and backups work well if we don't sync monitoring and backup passwords?

Reply:
I suspect, we will have a problems, as users will come from 1st cluster, but credentials in 2nd cluster peer relation databag will be old one. So we have to sync Juju secrert with values on the 1st cluster to avoid future issues.

Reply:
So, should we sync the monitor and backup password too?

Reply:
It has been addressed on [Discource](https://discourse.charmhub.io/t/charmed-mysql-how-to-clusterset-async-replication/10931) and will come here in the next automated PR.
I would like to merge this manual PR to have no open PRs before enabling automated sync (currently configured for mysql-k8s-operator repo only, but I wish to enable it for all SQL repos).